.github/workflows/memory-testing.yml

```diff
@@ -0,0 +1,47 @@
+---
+name: Memory Testing
+
+on:
+  pull_request:
+    paths:
+      - "crates/bitwarden-crypto/**"
+      - "crates/memory-testing/**"
+  push:
```

Comment:
❓ Is this needed if the intention is to catch these during code review?

Reply:
Not really, I based myself in the `rust-test.yml` which does run on push too, so I made it the same for consistency. But we could remove it and make it run only on PR

Reply:
Could catch non-PR scenarios too. I don't know why that'd happen but doesn't hurt.